MoltenVK/MoltenVK/GPUObjects/MVKImage.mm

```diff
@@ -763,10 +775,12 @@
 	_useMTLTextureView = _image->_canSupportMTLTextureView;
 
 	// If the view is identical to underlying image, don't bother using a Metal view
+	bool is3D = _image->_mtlTextureType == MTLTextureType3D;
 	if (_mtlPixelFormat == _image->_mtlPixelFormat &&
-		_mtlTextureType == _image->_mtlTextureType &&
+		(_mtlTextureType == _image->_mtlTextureType ||
+		 (_mtlTextureType == MTLTextureType2D || _mtlTextureType == MTLTextureType2DArray) && is3D) &&
```

Comment:
Can you fix the syntax warning that arises here regarding the order of && and ||, please?